specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json

```diff
@@ -948,6 +955,75 @@
         }
       }
     },
+    "AllowlistEntry": {
+      "required": [
+        "matchVariable",
+        "valueMatchOperator"
+      ],
+      "description": "Allows a request when the condition is satisfied.",
+      "properties": {
+        "matchVariable": {
+          "type": "string",
+          "enum": [
+            "RequestURI",
+            "RemoteAddr",
+            "RequestHeader"
+          ],
+          "description": "The variable on which we evaluate the allowlist condition",
+          "x-ms-enum": {
+            "name": "AllowlistEntryMatchVariable",
+            "modelAsString": true
+          }
+        },
+        "values": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Allowed values for the matchVariable"
+        },
+        "valueMatchOperator": {
+          "type": "string",
+          "enum": [
+            "Equals",
+            "Contains",
+            "StartsWith",
+            "EndsWith",
+            "IPMatch"
```

Comment:
Since you have the values as free-flowing strings, and a valueMatchOperator as Equals -- whats the need for IPMatch ?
Isnt that the same as string.equals ?

Reply:
IPMatch is required for matching both specific IPs as well as IP Ranges/CIDRs. So this won't be simple string matching. This is similar to the operator property in WebApplicationFirewallCustomRule.